ui/pages/confirmations/components/simulation-details/simulation-details.tsx

```diff
@@ -176,6 +183,7 @@ export const SimulationDetails: React.FC<SimulationDetailsProps> = ({
     return (
       <SimulationDetailsLayout
         inHeader={<LoadingIndicator />}
+        isTransactionsRedesign={isTransactionsRedesign}
```

Comment:
Rather than setting this directly and repeatedly passing it around as property between the various components here, and we can assume other components in future, would it be cleaner and more scalable to provide this via a React context defined in the root of the new confirmations, along with a hook such as `useIsTransactionRedesign`?

Then it can be seamlessly checked only where it's relevant, in this case directly in the `SimulationDetailsLayout` function?

Reply:
I've tried this but it made tests a bit more complicated. I prefer to use this pattern, in part also because I don't see us needing to reuse it a lot, so the context seems like an overkill. That said, if it becomes recurring, we should revisit this and give it another go.

Reply:
Not crucial now, as you say we won't need it alot, but I really appreciate the experimentation!

And unit tests should be as simple as mocking the new hook to return whatever value we need rather than passing a property, but we can worry about that in future.